src/Servers/Kestrel/Core/src/Internal/Infrastructure/WrappingStream.cs

```diff
@@ -107,6 +107,7 @@ public override int EndRead(IAsyncResult asyncResult)
         public override void EndWrite(IAsyncResult asyncResult)
             => _inner.EndWrite(asyncResult);
 
+        [Obsolete]
```

Comment:
You should match the params provided on base attribute.  Cc @jeffhandley

Reply:
Okay I updated to match https://github.com/dotnet/runtime/blob/329f0dbfa3a164ee2e3bf9f75b7641165995c799/src/libraries/System.Private.CoreLib/src/System/MarshalByRefObject.cs#L16. I gotta say, this is pretty viral.

Reply:
It's only viral for folks that want to hold the same bar as the framework, which I think we should be doing in .NET since we want to follow the same obsoletion roadmap across the board.  It would be bad, for example, if base shared framework obsoleted in release N then dropped support in N+1, but ASPnet dropped support in N.

For our customers it shouldn't be viral like this, since they can stop usage at the time we obsolete.